frontend/src/css/cat-theme.css

```diff
@@ -18,6 +23,7 @@ h1.green-divider:after {
 }
 
 h1 {
+  font-family: "Rubik", sans-serif !important;
```

Comment:
> Rubik is used for titles and headings, while Nunito Sans is used for sub-headings, buttons and paragraph text.

https://design.gccollab.ca/component/typography/

Does this mean we want all headings to use Rubrik?

Reply:
Actually, instead of both of these changes, what if you just did  this. Then all the headings would just be setup with the aurora stylesheet. 

```
i {
  font-family: "Arial", sans-serif;
}
```

Reply:
As discussed offline, there was a simpler way to do that. We just needed to put the new font styles before stylesheet in _index.html_ and remove all changes I did in _cat-theme.css_.

See commit #https://github.com/code-for-canada/project-thundercat/pull/105/commits/3ab22cae91eed0cd2d15dfbeb4dba99a474ded33